Model/Configuration.php

```diff
@@ -78,6 +84,11 @@ public function isEnabled()
         return (bool) $this->getConfig(self::MODULE_ENABLED_PATH);
     }
 
+	public function enable()
+	{
+		return $this->setConfig(self::MODULE_ENABLED_PATH, 1);
```

Comment:
Indentation weirdness here.

Edit: I'm going to stop flagging these. It looks like we have a mix of tabs and spaces all over this diff.